Makefile

```diff
@@ -367,7 +367,7 @@ update-mocks/%: %
 	@tools/update-mocks.sh "$*"
 
 unittest/%: %
-	$(VALGRIND) $(VALGRIND_TEST_ARGS) $*
+	$(VALGRIND) $(VALGRIND_TEST_ARGS) $* > /dev/null
```

Comment:
Mildly prefer redirecting to tmpfile and dumping the output if it fails, but that's more code...